src/net/rpc.rs

```diff
@@ -317,6 +317,12 @@ impl RPCPoxInfoData {
             ))?
             + 1;
 
+        let pox_3_first_cycle = burnchain
+            .block_height_to_reward_cycle(burnchain.pox_constants.pox_3_activation_height as u64)
+            .ok_or(net_error::ChainstateError(
+                "PoX-3 first reward cycle begins before first burn block height".to_string(),
+            ))?;
+
```

Comment:
Is this accurate? The `pox_2_first_cycle` calculation above adds a ` + 1` -- is that also needed here?

Reply:
Based off what I'm seeing after allowing epoch2.4 configuration in https://github.com/stacks-network/stacks-blockchain/pull/3705, it seems like this does need a `+ 1`. Not sure though, need feedback from someone more familiar.

Reply:
Yes, I think this should be +1. I believe the pox-3 contract gets deployed once the activation burn height is reached, so it actually ends up in the next block.

Reply:
Thanks @obycode! Fixed in https://github.com/stacks-network/stacks-blockchain/pull/3704/commits/5e6c36ed8076ae5ecf071e2c9a087d18fe8ea517